src/test/javascript/salvation.test.js

```diff
@@ -0,0 +1,39 @@
+const test = require('node:test');
+const assert = require('node:assert');
+const salvation = require('../../../target/javascript/salvation-v3.0.1.min.js');
+
+test('salvation initialization', () => {
+    assert.notStrictEqual(salvation.main, undefined);
+    try {
+        salvation.main();
+        assert.notStrictEqual(getErrorsForSerializedCSP, undefined);
+        assert.notStrictEqual(getErrorsForSerializedCSPList, undefined);
+    } catch (e) {
+        assert.fail('main function should not throw an exception', e.message);
```

Comment:
You don't need your own try/catch here; exceptions thrown in a test are treated as failures.